app/v3/controllers/requestParsers/validators/validations/TypeOfLossBusinessIdValidation.scala

```diff
@@ -32,4 +32,16 @@ object TypeOfLossBusinessIdValidation {
       case _                                                => List(RuleBusinessId)
     }
   }
+
+  def validate(typeOfLoss: TypeOfLoss, businessId: String): List[MtdError] = {
+    (typeOfLoss, businessId) match {
+      case (TypeOfLoss.`uk-property-non-fhl`, _)         => NoValidationErrors
+      case (TypeOfLoss.`uk-property-fhl`, _)             => NoValidationErrors
+      case (TypeOfLoss.`self-employment`, _)             => NoValidationErrors
+      case (TypeOfLoss.`self-employment-class4`, _)      => NoValidationErrors
+      case (TypeOfLoss.`foreign-property-fhl-eea`, _)    => NoValidationErrors
+      case (TypeOfLoss.`foreign-property`, _)            => NoValidationErrors
+      case _                                             => List(RuleBusinessId)
+    }
+  }
```

Comment:
Not really applicable and so should be removed since `RuleBusinessId` is mandatory and will be picked up by Play `Reads` validation. Will also need to remove the caller (`typeOfLossBusinessIdValidator` in the `CreateBGLossValidator` class).

Reply:
left that to be sortedin the Parser and validator subtask and left that to there and made this as a quick fix to avoid that